### PR TITLE
FEEL for image properties and adorners

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to #653 

Adds FEEL support for `alt`, `source` and adorners.

Demo: https://demo-653-adorners-images--camunda-form-playground.netlify.app/

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
This one is ready for review now. Feel free to have a look after public holidays :)

**Comment by pinussilvestrus** [comment]
Seems like the Carbonization is broken. I will have a look.

**Comment by pinussilvestrus** [comment]
@vsgoulart @Skaiir @christian-konrad @RomanKostka this one here is open for a while now. Just to ping it, there is no urgency as this won't make it much likely into the Modeler code freezes - no rush needed.

File: packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { useSingleLineTemplateEvaluation } from '../../../hooks';
+
+export default function TemplatedInputAdorner(props) {
+
+  const {
+    pre,
+    post
+  } = props;
+
+  const evaluatedPre = useSingleLineTemplateEvaluation(pre, { debug: true });
```
Comment (vsgoulart) [inline packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js:12]: is the `debug: true` expected to be here still?

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js:12]: Yeah it is a property to make `feelers` not throwing errors on invalid feel expressions. We could consider making this the default in the `useSingleLineTemplateEvaluation` hook though.

Wdyt?

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js:12]: Yeah it is a property to make `feelers` not throwing errors on invalid feel expressions. We could consider making this the default in the `useSingleLineTemplateEvaluation` hook though.

Wdyt?

Comment (vsgoulart) [reply packages/form-js-viewer/src/render/components/form-fields/parts/TemplatedInputAdorner.js:12]: I think it's ok to leave it as is, I was just wondering if this was something that was meant for use while testing only

**Review by vsgoulart — APPROVED** [review]
Code looks good, and it works well for me

**Comment by vsgoulart** [comment]
@pinussilvestrus Carbonisation failed, can you check if it's a real issue?

**Comment by pinussilvestrus** [comment]
Looks like the same error as https://github.com/bpmn-io/form-js/issues/674 again

**Review by christian-konrad — APPROVED** [review]
Like a charm!

**Pull request merged**